### ISSUE TITLE
(std -> core)::fmt::Display for DecodeError and DecodeKind

### ISSUE THREAD

**Issue by mriise** [ia0/data-encoding#42]
No need to feature flag fmt::Display for std as its in core.

```rust
impl core::fmt::Display for DecodeError {
    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
        write!(f, "{} at {}", self.kind, self.position)
    }
}
```

**Comment by ia0** [comment]
Thanks a lot for the report! That's indeed weird. Sending a fix.

**Comment by ia0** [comment]
It's published as version `2.3.2`.
